### PR TITLE
Add Yiddish language ruleset

### PR DESCRIPTION
**Pull request by yankl**
Added a ruleset for Yiddish, and added it to the default languages, as it's the only language so far here that uses this character set, and for any other languages that use these letters (e.g., Hebrew, Aramaic), it will be much better to have the Yiddish ruleset than nothing. 
I also added a test to `defaultRuleProvider` and ran the tests -- everything still passes.

### CODE REVIEW

**Review by florianeckerstorfer — APPROVED** [review]
Great, thank you

**Comment by florianeckerstorfer** [comment]
I think you need to run `php bin/generate-default.php` and commit the generated file

**Comment by yankl** [comment]
Sure, done! Since it's generated automatically I wasn't sure if I was supposed to include it in the PR.

**Comment by florianeckerstorfer** [comment]
@yankl Nice, thank you,

File: src/Slugify.php [hunk 1]
```diff
@@ -53,6 +53,7 @@ class Slugify implements SlugifyInterface
             // Languages are preferred if they appear later, list is ordered by number of
             // websites in that language
             // https://en.wikipedia.org/wiki/Languages_used_on_the_Internet#Content_languages_for_websites
+            'yiddish',
```
Comment (jorgsowa) [inline src/Slugify.php:56]: The whole list has an invalid order.

Comment (yankl) [reply src/Slugify.php:56]: Can you explain what you mean? What makes the order invalid?

Comment (jorgsowa) [reply src/Slugify.php:56]: Please look at the comment above the list in the code.

> list is ordered by number of websites in that language

Comment (yankl) [reply src/Slugify.php:56]: Yes, and it also says "Languages are preferred if they appear later" meaning the more popular languages should come last. And as you see, Russian, German and Polish appear near the bottom of the list here, while they're near the top of the list (and in the reverse order) on the wiki page. You're right that the order in general doesn't match. For example, whoever put Romanian at the bottom of the list made a mistake, as it's number 23 on the wiki page, way below German, for example. But I believe I correctly placed Yiddish at the top of the list, giving it very low priority, as it doesn't even appear on the list of most popular languages. And you're making this comment on this commit which only adds Yiddish to the list, hence my confusion. Are you claiming that Yiddish was incorrectly placed or just that the list as a whole is not in the stated order?

Comment (jorgsowa) [reply src/Slugify.php:56]: Exactly, the whole list is off the order.

**Pull request merged**